contrib/redhat/RPM-GPG-KEY-redhat-release

```diff
@@ -0,0 +1,34 @@
+pub   4096R/FD431D51 2009-10-22
```

Comment:
I think it'd be nice to fetch the key from the authoritative source instead of storing a copy here.